src/PatchOrchestrationApplication/NodeAgentNTService/src/Manager/TimerManager.cs

```diff
@@ -53,21 +55,24 @@ public void StartTimer()
             {
                 //System.Diagnostics.Debugger.Launch();
                 _eventSource.InfoMessage("Starting Timer.");
+
                 this.WaitForSettingsFile();
                 this.Clean();
 
                 this.LoadSettings();                
 
                 this.DisableWindowsUpdate();
                 
+                this.PostWUUpdateEventOnService();
+
                 this.ScheduleTimer();
             }
             catch (Exception e)
             {
                 string msg = string.Format("Exception while starting timer. {0}", e);
                 _eventSource.ErrorMessage(msg);
                 
-                this._nodeAgentSfUtility.ReportHealth("WindowsUpdateOperationResult",msg, HealthState.Warning, -1,
+                this._nodeAgentSfUtility.ReportHealth(WindowsUpdateSetting, msg, HealthState.Warning, -1,
```

Comment:
>WindowsUpdateSetting [](start = 54, length = 20)

This is incorrect right? WUUpdateSetting was only for telling about WU setting.

Reply:
I think this is all right. There are less chances of ScheduleTimer throwing any exception which is not caused by any setting issue.